liquidity-rewards/.env.example

```diff
@@ -0,0 +1,8 @@
+# Used to deploy to testnets
+INFURA_API_KEY=''
+
+# Used to deploy to testnets
+HDWALLET_MNEMONIC=''
+
+# Used for verifying contracts on Etherscan
+ETHERSCAN_API_KEY=''
```

Comment:
@hhio618 Should the 4 environment variables in `liquidity-rewards/migrations/1_deploy_contract.js` also be added here?

Reply:
That's right, thanks for reminding me!